Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -291,10 +299,17 @@ void HTMLAttachmentElement::updateProgress(const AtomString& progress)
     bool validProgress = false;
     float value = progress.toFloat(&validProgress);
     if (validProgress && std::isfinite(value)) {
+        if (value == 0.0) {
```

Comment:
Do we want a `areEssentiallyEqual` here or is it really really 0?

Reply:
I think we really want 0.0, as we just want to check for `progress="0"`, and that matches the check in the legacy attachment code.

Reply:
Can we just do `!value` then?

Reply:
I personally hate that for numbers, but fiiine, if that's the preferred style... 😬

Reply:
It is 😅 

https://webkit.org/code-style-guidelines/#zero-comparison